spec/centos_spec.rb

```diff
@@ -80,13 +80,10 @@
 
     it 'Install os shipped package' do
       expect(chef_run).to install_package('mariadb-server')
+      expect(os_package).to notify('directory[/var/log/mysql]').to(:create).immediately
       expect(os_package).to notify('service[mysql]').to(:start).immediately
     end
 
-    it 'Create Log directory' do
```

Comment:
I think this is still valid, you could add a .with(....)  to make it more specific.